config/default/crds/patches/crd_status_condition_patch.yaml

```diff
@@ -1,3 +1,18 @@
+- op: replace
```

Comment:
should the name of this file change now that it's both patches?

Reply:
good point

Reply:
renamed